src/hooks/useTransactions.ts

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { AppState } from '@rainbow-me/redux/store';
+import { ethereumUtils } from '@rainbow-me/utils';
+
+export default function useTransactions() {
+  const transactions = useSelector(({ data }: AppState) => data.transactions);
+
+  const getTransactionByHash = useCallback(
```

Comment:
want to confirm that the txn hash format is what you expect, even if it comes from zerion and is parsed thru our parsers.

iirc, we have a pattern where we would parse txn hashes as `{txnhash}-0` or `{txnhash}-1`